src/expr/skolem_manager.h

```diff
@@ -177,8 +177,14 @@ class SkolemManager
    * @return The skolem function.
    */
   Node mkSkolemFunction(SkolemId id, Node cacheVal = Node::null());
-  /** Same as above, with multiple cache values */
-  Node mkSkolemFunction(SkolemId id, const std::vector<Node>& cacheVals);
+  /**
+   * Same as above, with multiple cache values.
+   * @param id The identifier of the skolem function
+   * @param cacheVals A cache value.
+   * @return The skolem function.
+   */
+  Node mkSkolemFunction(SkolemId id,
```

Comment:
This version is used quite frequently throughout the code.

One alternative I have considered is to *always* use vectors, so that this SEXPR detail is an internal detail of this class only.